publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RabbitMqProperties.java

```diff
@@ -494,6 +495,13 @@ public void shutdownCompleted(ShutdownSignalException cause) {
                     } else {
                         log.error("Shutdown is NOT initiated by application.");
                         log.error(cause.getMessage());
+                        if(rabbitConnection != null || !rabbitConnection.isOpen()) {
+                            try {
+                                rabbitConnection.close();
+                            } catch (IOException e) {
+                                e.printStackTrace();
```

Comment:
Use log.error with a comment clearly stating that this is due to an attempt to close the connection.